File: python_programming_basics/conditional_statements_advanced/e_06_operations_between_numbers.py
first_number = int(input())
second_number = int(input())
operator = input()
result = 0

if operator == "+":
    result = first_number + second_number
    print(f"{first_number} {operator} {second_number} = {result} - {'even' if result % 2 == 0 else 'odd'}")

elif operator == "-":
    result = first_number - second_number
    print(f"{first_number} {operator} {second_number} = {result} - {'even' if result % 2 == 0 else 'odd'}")

elif operator == "*":
    result = first_number * second_number
    print(f"{first_number} {operator} {second_number} = {result} - {'even' if result % 2 == 0 else 'odd'}")

elif operator == "/":
    if second_number == 0:
        print(f"Cannot divide {first_number} by zero")

    else:
        result = first_number / second_number
        print(f"{first_number} / {second_number} = {result:.2f}")

elif operator == "%":
    if second_number == 0:
        print(f"Cannot divide {first_number} by zero")

    else:
        result = first_number % second_number
        print(f"{first_number} % {second_number} = {result}")

File: JavaScript Basics/03. Conditional Statements Advanced/e-06-Operations-Between-Numbers.js
function calculate(N1, N2, operator) {
    let result;

    if (operator === "+") {
        result = N1 + N2;
        let evenOrOdd = result % 2 === 0 ? "even" : "odd";
        console.log(`${N1} + ${N2} = ${result} - ${evenOrOdd}`);
    } 
    else if (operator === "-") {
        result = N1 - N2;
        let evenOrOdd = result % 2 === 0 ? "even" : "odd";
        console.log(`${N1} - ${N2} = ${result} - ${evenOrOdd}`);
    } 
    else if (operator === "*") {
        result = N1 * N2;
        let evenOrOdd = result % 2 === 0 ? "even" : "odd";
        console.log(`${N1} * ${N2} = ${result} - ${evenOrOdd}`);
    } 
    else if (operator === "/") {
        if (N2 === 0) {
            console.log(`Cannot divide ${N1} by zero`);
        } else {
            result = (N1 / N2).toFixed(2);
            console.log(`${N1} / ${N2} = ${result}`);
        }
    } 
    else if (operator === "%") {
        if (N2 === 0) {
            console.log(`Cannot divide ${N1} by zero`);
        } else {
            result = N1 % N2;
            console.log(`${N1} % ${N2} = ${result}`);
        }
    }
}

calculate(10, 12, "+");